scripts/build-htslib.sh

```diff
@@ -0,0 +1,15 @@
+set -eu
+
+PATCH_DIR="$1"
+
+LIBHTS_SOVERSION=${LIBHTS_SOVERSION-3}
+
+./configure CFLAGS=-DCURL_STATICLIB
+
+# apply patches
+patch Makefile "${PATCH_DIR}/makefile.staticlink.patch"
+patch config.mk "${PATCH_DIR}/config.mk.staticlink.patch"
```

Comment:
There are a couple of additional source patches (for .h) files that should probably be applied...
Note that two apply to 1.15.1, and only one applies to 1.16 (as the other change already appears in 1.16), not sure whether they were previously provided to you or not.
files are named...
htslib.1.15.1.hts_defs.h.patch
htslib.1.15.1.vcf.h.patch

and htslib.1.15.1.vcf.h.patch can also be applied to 1.16

Reply:
> There are a couple of additional source patches (for .h) files that should probably be applied...

I wasn't explicitly sent these additional patches. You have multiple branches in TileDB-VCF. Could you please send me links to the exact patch files I should apply?

> and htslib.1.15.1.vcf.h.patch can also be applied to 1.16

Also, just to confirm, my understanding is that the plan is to first build htslib 1.15.1 for use with tiledb-vcf-feedstock, and then later update to 1.16